LibraBFT/Impl/Consensus/RoundManager.agda

```diff
@@ -92,14 +96,14 @@ module ensureRoundAndSyncUpM
 
   step₀ = do
     currentRound ← use (lRoundState ∙ rsCurrentRound)
-    if ⌊ messageRound <? currentRound ⌋
+    ifM messageRound <? currentRound
       then ok false
       else step₁
   step₁ =
         syncUpM now syncInfo author helpRemote ∙?∙ λ _ → step₂
   step₂ = do
           currentRound' ← use (lRoundState ∙ rsCurrentRound)
-          if not ⌊ messageRound ≟ℕ currentRound' ⌋
+          ifM not ⌊ messageRound ≟ℕ currentRound' ⌋
```

Comment:
Because `not` has type `Bool -> Bool`. We could rename in the `Prelude` and define a version of `not` that it accepts any type with a `ToBool` instance, which would remove the need for the floor symbols.

Reply:
I think it is a good idea to do as you suggest and avoid using floor in the code (for consistency).